examples/polyglot-zipkin/java-wildfly-swarm/src/main/java/org/hawkular/apm/example/swarm/dao/UserDAO.java

```diff
@@ -60,41 +58,51 @@ public UserDAO(Brave brave) {
     }
 
     private void init() {
-        PreparedStatement preparedStatement = session.prepare("DROP KEYSPACE IF EXISTS " + KEYSPACE).enableTracing();
-        executeWithClientSpan(preparedStatement.bind());
+        BoundStatement boundStatement = session.prepare("DROP KEYSPACE IF EXISTS " + KEYSPACE).bind();
+        executeWithClientSpan(boundStatement);
 
-        preparedStatement = session.prepare("CREATE KEYSPACE IF NOT EXISTS " + KEYSPACE + " WITH REPLICATION = " +
-                "{'class' : 'SimpleStrategy', 'replication_factor' : 1}").enableTracing();
-        executeWithClientSpan(preparedStatement.bind());
+        boundStatement = session.prepare("CREATE KEYSPACE IF NOT EXISTS " + KEYSPACE + " WITH REPLICATION = " +
+                "{'class' : 'SimpleStrategy', 'replication_factor' : 1}").bind();
+        executeWithClientSpan(boundStatement);
 
-        preparedStatement = session.prepare("USE " + KEYSPACE).enableTracing();
-        executeWithClientSpan(preparedStatement.bind());
+        boundStatement = session.prepare("USE " + KEYSPACE).bind();
+        executeWithClientSpan(boundStatement);
 
-        preparedStatement = session.prepare("CREATE TABLE " + KEYSPACE + "." + TABLE +
-                "(id text PRIMARY KEY, name text)").enableTracing();
-        executeWithClientSpan(preparedStatement.bind());
+        boundStatement = session.prepare("CREATE TABLE " + KEYSPACE + "." + TABLE +
+                "(id text PRIMARY KEY, name text)").bind();
+        executeWithClientSpan(boundStatement);
     }
 
     public User createUser(User user) {
         user.setId(UUID.randomUUID().toString());
 
-        Statement statement = QueryBuilder.insertInto(KEYSPACE, TABLE)
-                .value("id", user.getId())
-                .value("name", user.getName())
-                .enableTracing();
+        /**
+         * With query builder tracing is not reported in C*, but
+         * it shows complete query with parameters when calling statement.toString();
+         */
+//        Statement statement = QueryBuilder.insertInto(KEYSPACE, TABLE)
```

Comment:
I would like to keep this comments, they show that tracing is not propagated to C* tracer  if somebody uses `QueryBuilder`.